deployment/deployment.json

```diff
@@ -9758,5 +9758,35 @@
         }
       }
     }
+  },
+  "eigenlayer": {
+    "schema": "non-standard",
```

Comment:
we should make this say generic, the name here is actually used on our ingestion pipeline. And it would be confused otherwise (i think)